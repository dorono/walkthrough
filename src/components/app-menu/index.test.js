import React from 'react';
import {shallow} from 'enzyme';
import AppMenu from './index';

describe('AppMenu', () => {
    it('should render without errors', () => {
        shallow(<AppMenu />);
    });

    it('should match snapshot', () => {
        expect(shallow(<AppMenu />)).toMatchSnapshot();
    });
});
