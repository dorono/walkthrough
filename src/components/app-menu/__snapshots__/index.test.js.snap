// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppMenu should match snapshot 1`] = `
<div
  className="root"
>
  <NavLink
    activeClassName="active"
    ariaCurrent="true"
    className="link"
    isActive={[Function]}
    to="/dblocks"
  >
    BLOCKS
  </NavLink>
  <NavLink
    activeClassName="active"
    ariaCurrent="true"
    className="link"
    isActive={[Function]}
    to="/chains"
  >
    CHAINS
  </NavLink>
</div>
`;
